craystack/codecs_test.py

```diff
@@ -166,7 +166,7 @@ def test_categorical_new():
     rng = np.random.RandomState(2)
     precision = 4
     shape = (20, 3, 5)
-    weights = rng.random((np.prod(shape), 4))
+    weights = rng.random((np.prod(shape), 4)) + 1
```

Comment:
The lack of the `+ 1` was sometimes causing the test to fail, as some weights would be quantized to `0` due to low precision.